xesmf/tests/test_frontend.py

```diff
@@ -952,3 +952,10 @@ def test_spatial_averager_mask():
     savg = xe.SpatialAverager(dsm, [poly], geom_dim_name='my_geom')
     out = savg(dsm.abc)
     assert_allclose(out, 2, rtol=1e-3)
+
+
+def test_densify_polys():
```

Comment:
Do we test that segments over 1 deg raise a warning ?

Reply:
I'll add the test to make sure the warning is raised.